src/multicellular_stage/MulticellularSpecies.cs

```diff
@@ -188,6 +199,54 @@ public override float GetPredationTargetSizeFactor()
         return totalOrganelles;
     }
 
+    public void SetupWorldEntities(IWorldSimulation worldSimulation)
+    {
+        ((MicrobeVisualOnlySimulation)worldSimulation).CreateVisualisationColony(this);
+    }
+
+    public bool StateHasStabilized(IWorldSimulation worldSimulation)
+    {
+        return true;
+    }
+
+    public Vector3 CalculatePhotographDistance(IWorldSimulation worldSimulation)
+    {
+        float radius = 0.0f;
+
+        Vector3 center = Vector3.Zero;
+
+        int count = Cells.Count;
+        for (int i = 0; i < count; ++i)
+        {
+            center += Hex.AxialToCartesian(Cells[i].Position);
+        }
+
+        center /= count;
+
+        foreach (var entity in worldSimulation.EntitySystem)
+        {
+            if (!entity.Has<CellProperties>())
+                continue;
+
+            ref var cellProperties = ref entity.Get<CellProperties>();
+
+            // This uses the membrane as radius is not set as the physics system doesn't run
+            if (!cellProperties.IsMembraneReady())
+                throw new InvalidOperationException("Microbe doesn't have a ready membrane");
+
+            var cellRadius = cellProperties.CreatedMembrane!.EncompassingCircleRadius;
+
+            var farthestPoint = entity.Get<WorldPosition>().Position.DistanceTo(center) + cellRadius;
```

Comment:
Same squared distance comparison optimization could be done here as well I think.